content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+```shell
+Error from server (Forbidden): error when creating "limit-range-pod-2.yaml": pods "busybox2" is forbidden: [maximum cpu usage per Pod is 2, but limit is 2400m., maximum memory usage per Pod is 2Gi, but limit is 2306867200.]
+```
+
+```shell
+kubectl get  po/busybox1  -n limitrange-demo -o json | jq ".spec.containers[].resources.limits.memory" 
+"200Mi"
+"900Mi"
+"200Mi"
+"900Mi"
+```
+
+We can also learn from the previous example thant LimitRange validations occurs only at Pod Admission stage, not on Running pods.
+
+
+## Storage LimitRanges
+
+You can enforce  min/max  size  of [storage resources](/docs/concepts/storage/persistent-volumes/) that can be requested by each PersistentVolumeClain in a namespace using a LimitRange.
+
+{{< codenew file="admin/resource/storagelimits.yaml" >}}
+
+Apply the YAML using `kubectl create`.
+
+```shell
+kubectl create -f https://k8s.io/examples/admin/resource/storagelimits.yaml -n limitrange-demo 
+```
+
+```shell
+limitrange/storagelimits created
+```
+Describe the created object, 
+
+```shell
+kubectl describe limits/storagelimits  
+```
+the output should look like 
+
+```shell
+Name:                  storagelimits
+Namespace:             limitrange-demo
+Type                   Resource  Min  Max  Default Request  Default Limit  Max Limit/Request Ratio
+----                   --------  ---  ---  ---------------  -------------  -----------------------
+PersistentVolumeClaim  storage   1Gi  2Gi  -                -              -
+```
+
+{{< codenew file="admin/resource/pvc-limit-lower.yaml" >}}
+
+```shell
+kubectl create -f https://k8s.io/examples/admin/resource//pvc-limit-lower.yaml -n limitrange-demo
+``` 
+
+While creating a PVC with `requests.storage` lower than the Min value in the LimitRange, an Error thrown by the server 
+
+```shell
+Error from server (Forbidden): error when creating "pvc-limit-lower.yaml": persistentvolumeclaims "pvc-limit-lower" is forbidden: minimum storage usage per PersistentVolumeClaim is 1Gi, but request is 500Mi.
+```
+
+Same behaviour is noted if  the `requests.storage` is greater than the Max value  in the LimitRange 
+
+{{< codenew file="admin/resource/pvc-limit-greater.yaml" >}}
+
+```shell
+kubectl create -f https://k8s.io/examples/admin/resource/pvc-limit-greater.yaml -n limitrange-demo
+``` 
+
+```shell
+Error from server (Forbidden): error when creating "pvc-limit-greater.yaml": persistentvolumeclaims "pvc-limit-greater" is forbidden: maximum storage usage per PersistentVolumeClaim is 2Gi, but request is 5Gi.
+```
+
+
+## Limits/Requests Ratio 
+
+In a LimitRange object , when `MaxLimitRequestRatio` if specified, the named resource must have a request and limit that are both non-zero where limit divided by request is less than or equal to the enumerated value
+
+Let's create the following `LimitRange` to ensure memory limit should be at most the Double of memory request for any pod in our namespace 

```

Comment:
line 330, check the API reference for correct object names, If `LimitRangeItem.maxLimitRequestRatio` is set in the `LimitRangeSpec`, ...
line 332, possible clean up, not clear on the content:
Create a `LimitRange` with the memory limit set at twice the amount of the memory [permitted] for any pod in the namespace.

Reply:
rephrased. Thanks